doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -605,6 +594,9 @@ name "/OF160/"
 
  Beagle's results do not change, no matter how often the user chooses to
  pause and resume the measurements.
+ In fact, the cooling of the CPU during exoneration turns out to be a real
```

Comment:
Talking of problems in the requirements seems odd to me. Why not omitting the whole sentence? For example:

“ Beagle's results do not change, no matter how often the user chooses to
  pause and resume the measurements. Beagle assures approximately constant measurement conditions, e.g. by heating up the CPU with load before starting measurements when resuming.”